docs/build-repro.md

```diff
@@ -127,3 +127,47 @@ jq '.' metadata.json
   "containerimage.digest": "sha256:..."
 }
 ```
+
+### Reproducing the pinned dependencies
+
+<!-- TODO: s/master/v0.11/ after the release -->
+Reproducing the pinned dependencies is supported in the master branch of BuildKit.
+
+e.g.,
+```bash
+buildctl build --frontend dockerfile.v0 --local dockerfile=. --local context=. --source-policy-file Dockerfile.pol
+```
+
+An example `Dockerfile.pol`:
```

Comment:
It might be safer to keep the filename to be decoupled from the Dockerfile frontend.
e.g., `policy.json`.


The `Dockerfile.pol` name might be reused for the human-friendly representation and Dockerfile-specific representation of the policy.

Reply:
Yes there is also some linguist rules that would highlight `Dockerfile.*` files as Dockerfile syntax.